### PR TITLE
Enhancement/split tx consumer from pbft

### PR DESCRIPTION
**Pull request by VargaElod23**

### CODE REVIEW

File: services/explorer-indexer/src/modules/live-sync/live.syncer.service.ts [hunk 1]
```diff
@@ -42,10 +45,12 @@ export default class LiveSyncerService {
     private readonly dagService: DagService,
     private readonly pbftService: PbftService,
     private readonly txService: TransactionService,
-    private readonly graphQLConnector: GraphQLConnectorService
+    private readonly graphQLConnector: GraphQLConnectorService,
+    private readonly historicalSyncerService: HistoricalSyncService
   ) {
     this.logger.log('Starting NodeSyncronizer');
     this.isWsConnected = false;
+    this.queueCache = new QueuePopulatorCache(this.pbftsQueue, 10);
```
Comment (rattrap) [inline services/explorer-indexer/src/modules/live-sync/live.syncer.service.ts:53]: If we use this in both modules, maybe we can move it to a separate, `common`, module

File: services/explorer-indexer/src/modules/live-sync/live.syncer.service.ts [hunk 2]
```diff
@@ -167,15 +172,28 @@ export default class LiveSyncerService {
...
-          );
-          this.logger.debug(`Pushed ${formattedNumber} into PBFT sync queue`);
+          if (this.historicalSyncerService.isRunning) {
+            await this.queueCache.add({
+              name: QueueJobs.NEW_PBFT_BLOCKS,
+              data: {
+                pbftPeriod: formattedNumber,
+                type: SyncTypes.LIVE,
+              } as QueueData,
+              JobKeepAliveConfiguration,
```
Comment (rattrap) [inline services/explorer-indexer/src/modules/live-sync/live.syncer.service.ts:182]: Seems like a duplicate block. We do these in both cases

**Review by rattrap — CHANGES_REQUESTED** [review]

File: services/explorer-indexer/src/modules/live-sync/live.syncer.service.ts (hunk 2 above)
Comment (VargaElod23) [reply services/explorer-indexer/src/modules/live-sync/live.syncer.service.ts:182]: Yes, however, we do add the period to different places.

**Review by rattrap — APPROVED** [review]
LGTM

**Pull request merged**